caniusepython3/overrides.json

```diff
@@ -21,9 +23,12 @@
     "flask-principal": "http://pythonhosted.org/Flask-Principal/changelog.html",
     "functools32": "https://pypi.python.org/pypi/functools32",
     "futures": "https://docs.python.org/3/library/concurrent.futures.html",
+    "hash-ring": "https://github.com/Doist/hash_ring/blob/master/tox.ini",
```

Comment:
This actually isn't accurate as they added the test support after their latest release.
```suggestion
```